### ISSUE TITLE
How would I add .HasConversion to .HasColumnType("char(45)")

### ISSUE THREAD

**Issue by PETERHUP** [ReversePOCO/EntityFramework-Reverse-POCO-Code-First-Generator#712]
I am using a legacy database that uses Char instead of Varchar for all of its string columns. I have found some code that works perfectly for trimming those Char fields:
```cs
.HasConversion(new ValueConverter<string, string>(v => v.TrimEnd(), v => v.TrimEnd()))
```
but unsure how to get the generated code to add this for SQL fields that are of type char in the .tt file.

**Example of Generated Configuration:**
```cs
builder.Property(x => x.Addr2)
  .HasColumnName(@"Addr2")
  .HasColumnType("char(45)")
  .IsRequired()
  .IsFixedLength()
  .IsUnicode(false)
  .HasMaxLength(45);
```

**What I need generated:**
```cs
builder.Property(x => x.Addr2)
  .HasColumnName(@"Addr2")
  .HasColumnType("char(30)")
  .HasConversion(new ValueConverter<string, string>(v => v.TrimEnd(), v => v.TrimEnd()))
  .IsRequired()
  .IsFixedLength()
  .IsUnicode(false)
  .HasMaxLength(30);
```

**Comment by PETERHUP** [comment]
So, to make this work I modified the `protected override void SetupConfig(Column c)` , just after the `if (!string.IsNullOrEmpty(c.SqlPropertyType)) { }`

I added the following to make it work
```cs
if (c.DbName.ToLower() != "clientno" && c.DbName.ToLower() != "emplno" && c.SqlPropertyType == "char")
{
    sb.Append(".HasConversion(new ValueConverter<string, string>(v => v.TrimEnd(), v => v.TrimEnd()))");
}
```

**Comment by sjh37** [comment]
I'll add an option that the developer can choose to trim char fields or not in the `settings.tt` file.

**Comment by sjh37** [comment]
Done. This will be in the next release. Thanks @PETERHUP

**Comment by sjh37** [comment]
Now released in v3.5.0
